### DISCUSSION TITLE
Extra window popup on export

### DISCUSSION THREAD

**Discussion by QiangF** [kjambunathan/org-mode-ox-odt#192] [General]
In org-odt--translate-clickable-images, (pop-to-buffer (current-buffer)) create an extra window.
I find it's distracting. Is there any use for that?
Thank you!

**Comment by kjambunathan** [comment]
Most likely a.debug code, if it is in that function.

It is possible that I forgot to remove it.

Let me see tomorrow.

Meanwhile you can comment out that part.
